packages/utils/src/fallbackSubgraph.ts

```diff
@@ -180,13 +188,17 @@ export class FallbackSubgraph<T extends SubgraphSdk> {
    * @returns Subgraph sync records for each subgraph.
    */
   public async sync(_latestBlock?: number): Promise<SubgraphSyncRecord[]> {
+    // If the latest sync was within 5s, do not requery
```

Comment:
lol super nit but this comment should be
`// If the latest sync was within SYNC_CACHE_TTL, do not requery`
haha